pkg/gateway/operations/getobject.go

```diff
@@ -28,6 +28,10 @@ func (controller *GetObject) RequiredPermissions(_ *http.Request, repoID, _, pat
 }
 
 func (controller *GetObject) Handle(w http.ResponseWriter, req *http.Request, o *PathOperation) {
+	// TODO(barak): do we like to reject tagging or return empty tagging?
```

Comment:
Pick one.  I'd say reject, but I _definitely_ don't want get-tags to return the object contents -- which AFAICT the current code does.